### PR TITLE
Tag IntervalWavelets v0.0.4

### PR DESCRIPTION
**Pull request by robertdj**
The new version *is* compatible with v0.5

### CODE REVIEW

File: IntervalWavelets/versions/0.0.4/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Wavelets
+Compat
```
Comment (tkelman) [inline IntervalWavelets/versions/0.0.4/requires]: should declare a minimum version dependency that implements the features you need

Comment (robertdj) [inline IntervalWavelets/versions/0.0.4/requires]: It should be fixed now.

Comment (tkelman) [inline IntervalWavelets/versions/0.0.4/requires]: yep, looks good, though be sure to make the same change at https://github.com/robertdj/IntervalWavelets.jl/blob/master/REQUIRE so it'll get picked up for the next tag too

Comment (robertdj) [inline IntervalWavelets/versions/0.0.4/requires]: I've just pushed an update.

**Pull request merged**